test/Assert.t.sol

```diff
@@ -13,27 +13,16 @@ contract TestAsserts is Test, HelperAssert {
         setPlatform(address(new PlatformTest()));
     }
 
-    function test_TFail() public {
-        string memory reason = "Testing assertion";
-
-        vm.expectEmit(true, true, true, true);
-        emit AssertFail("Testing assertion");
-        vm.expectRevert(PlatformTest.TestAssertFail.selector);
-        t(false, reason);
-    }
-
-    function test_TSuccess() public {
-        string memory reason = "Testing assertion";
-        t(true, reason);
-    }
-
     function test_HelperAssert_t_true() public {
         t(true, "t does not revert for true"); 
     }
 
     function test_HelperAssert_t_false() public {
-        vm.expectRevert(PlatformTest.TestAssertFail.selector);
-        t(false, "t does not revert for true"); 
+        string memory reason = "t does not revert for true";
```

Comment:
this text does not make sense for the test (testing false). i suggest we change it to some generic text that is easily copy/pasteable